Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
```

Comment:
No it would mean they were using, for example, buildpath/"foo" as the prefix. It's already protected against multiple packing, though, due to the .npmrc check.